<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Robocon夏令营第一组的遥控器</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>166</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>波特率</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>加\n</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="serialNumberComboBox"/>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="addEnterComboBox">
      <item>
       <property name="text">
        <string>是</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>否</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="BaudRateComboBox">
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9600</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>串口号</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="SwitchOnSerialPortPushButton">
      <property name="text">
       <string>打开串口</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="scanSerialPortPushButton">
      <property name="text">
       <string>扫描串口</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="receiveTextEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>321</width>
     <height>391</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code Retina</family>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="sendTextLineEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>480</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sendMessagePushButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>510</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearReceiveTextPushButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>430</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>清空接收</string>
   </property>
  </widget>
  <widget class="QPushButton" name="WCommandPushButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>420</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>W</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SCommandPushButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>480</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ACommandPushButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>480</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DCommandPushButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>480</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>D</string>
   </property>
  </widget>
  <widget class="QPushButton" name="emergencyStopCommandPushButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>540</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>紧急停止</string>
   </property>
  </widget>
  <widget class="QPushButton" name="QCommandPushButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>420</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Q</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ECommandPushButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>420</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>E</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lockerPushButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>180</y>
     <width>171</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>锁定</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="saveConfigPushButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>367</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>保存配置</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
